app/src/screens/SplashScreen.tsx

```diff
@@ -39,7 +39,7 @@ const SplashScreen: React.FC = ({}) => {
         navigation.navigate('Launch');
       }
     }, 1000);
-  }, [loadSecret, loadPassportData, navigation]);
+  }, [navigation]);
```

Comment:
why is it ok for hasStoredSecret to not be in this?

Reply:
imported functions or consts are stable (aka they dont ever change) so they don't need to be in the dependency arrays of hooks

`loadSecret` and `loadPassportData` were wrongly put there in the first lace

Reply:
oops i thought i was from a hook